demos/react-demo/src/pages/svg.tsx

```diff
@@ -5,9 +5,137 @@ import { Theme } from '@mui/material/styles';
 // import camelCase from 'camelcase';
 import Breaker from '@brightlayer-ui/icons-svg/breaker.svg';
 
-const meta = require('@brightlayer-ui/icons-mui/index.json');
+const svgIcons = [
```

Comment:
Scale this array down to include only the data that is necessary to render the icons on the screen (there is a bunch of unnecessary data in here that just makes the file harder to read). Or pull the data from the actual meta file (using something like array.filter to filter by the names we want).